src/pybind/mgr/diskprediction_cloud/common/__init__.py

```diff
@@ -29,7 +29,7 @@ class TimeoutError(Exception):
     pass
 
 
-def timeout(seconds=10, error_message=os.strerror(errno.ETIME)):
+def timeout(seconds=10, error_message=os.strerror(errno.ETIMEDOUT)):

```

Comment:
I don't get the sense of using OS `errno` codes for getting an i18n'ed app error string for an exception that doesn't even inherit from `OSError`, whose value is never checked, and while everything else is not i18n'ed either.

I'd rather remove the whole `error_message=os.strerror(errno.ETIMEDOUT)` line and put the default message in the exception:
```py
class TimeoutError(Exception):
  def __init__(self, msg="Something kinda timed out somewhere"):
    super(TimeoutError, self).__init__(msg)
```

Reply:
And how different is this from a less invasive version?
```py
def timeout(seconds=10, error_message="Something kinda timed out somewhere"):
```

Reply:
@wjwithagen the difference is all about readability. mainly, explicit is better than implicit.

- better readability: this `timeout()` function is never called with the `error_message` parameter.
- and better readability: msg is not connected to a certain errno.
- also better readability: TimeoutError is always constructed with this `error_message` so, let's just make this explicit.

Reply:
i created a more invasive version at #31176